commands/acronyms.js

```diff
@@ -0,0 +1,48 @@
+const Command = require("../base/Command");
+
+
+/**
+ * The list of acronyms defined in data/acronym.json was transposed from the SWGoH forum. Any update there will need to make it's way into here.
+ * //https://forums.galaxy-of-heroes.starwars.ea.com/discussion/154048/guide-to-the-acronyms-and-terms-of-star-wars-galaxy-of-heroes-swgoh
+ */
+class Acronyms extends Command {
+    constructor(client) {
+        super(client, {
+            name: "acronyms",
+            category: "Misc",
+            aliases: ["acr", "acronym"],
+        });
+    }
+
+    async run(client, message, [ acronym ], options) { // eslint-disable-line no-unused-vars
+        const acronymsLookup = client.acronyms;
+        const acronyms = Object.keys(acronymsLookup);
+
+        if (!acronym.length) {
+            return message.channel.send(message.language.get("COMMAND_ACRONYMS_INVALID", acronyms.join(", ")));
+        }
+
+        const lookupList = acronym.split("|").map(a => a.toLowerCase());
```

Comment:
To get rid of extra spaces before or after that'd make it not match
```suggestion
        const lookupList = acronym.split("|").map(a => a.trim().toLowerCase());
```